### PR TITLE
Add release notes for KubeDB v2022.12.28

### PR DESCRIPTION
**Pull request by heheh13**
Signed-off-by: Mehedi Hasan <mehedi.hasan@appscode.com>

### CODE REVIEW

File: content/post/kubedb-v2022.12.28/index.md [hunk 1]
```diff
@@ -0,0 +1,389 @@
...
+- opensearch
+- percona
+- percona-xtradb
+- postgresql
+- redis
+- schema-manager
+---
+
+We are pleased to announce the release of [KubeDB v2022.12.28](https://kubedb.com/docs/v2022.12.28/setup/). This post lists all the major changes done in this release since the last release. 
+The release was mainly focused on updating documentations , writing test and bug fixes and improvements. Aside there was some major features like **kafka,** **ProxySQL: Autoscaler,Monitoring,MariaDB and Percona-XtraDB backend** support. Also, new versions for `ElasticSearch` , `MariaDB`, `PgBouncer`, `Kibana` are available.
```
Comment (rakibulhossain) [inline content/post/kubedb-v2022.12.28/index.md]: -> Also, new versions for `ElasticSearch` , `MariaDB`, `PgBouncer`, `Kibana` are available.
Add PostgreSQL

**Review by rakibulhossain — DISMISSED** [review]
LGTM!

File: content/post/kubedb-v2022.12.28/index.md [hunk 2]
```diff
@@ -0,0 +1,389 @@
...
+- opensearch
+- percona
+- percona-xtradb
+- postgresql
+- redis
+- schema-manager
+---
+
+We are pleased to announce the release of [KubeDB v2022.12.28](https://kubedb.com/docs/v2022.12.28/setup/). This post lists all the major changes done in this release since the last release. 
+The release was mainly focused on updating documentations , writing test and bug fixes and improvements. Aside there was some major features like **kafka,** **ProxySQL: Autoscaler,Monitoring,MariaDB and Percona-XtraDB backend** support. Also, new versions for `ElasticSearch` , `MariaDB`,`PostgreSQL`, `PgBouncer`, `Kibana` are available.
```
Comment (raihankhan) [inline content/post/kubedb-v2022.12.28/index.md:29]: typo: kafka > Kafka

File: content/post/kubedb-v2022.12.28/index.md [hunk 3]
```diff
@@ -0,0 +1,389 @@
...
+### Bug Fix and improvement:
+- Fixed Transfer Leadership issue on raft switchover.
+- Fixed issue with Single User mode on raft sidecar.
+- Fix issue with pre-conflict error detection in Logical Replication:
+
+## Kibana
+### New Version Support: 
+We have added support for the latest [Kibana version 8.5.2](https://www.elastic.co/guide/en/kibana/current/release-notes-8.5.2.html) which is compatible with Elasticsearch 8.5.2. If you have an Elasticsearch cluster with version `xpack-8.5.2` provisioned with KubeDB, you can apply the following yaml to provision TLS secured Kibana 8.5.2 standalone cluster. Just referring to the database in the dashboard yaml is enough as the operator provisions compatible Kibana version with the Elasticsearch cluster.
+
+```
```
Comment (raihankhan) [inline content/post/kubedb-v2022.12.28/index.md:360]: Set the yaml tag in code blocks - 
```
```yaml
```

**Review by raihankhan — CHANGES_REQUESTED** [review]

File: content/post/kubedb-v2022.12.28/index.md (hunk 2 above)
Comment (ArnobKumarSaha) [inline content/post/kubedb-v2022.12.28/index.md]: All the topics should be like this `** <topic> **`.

File: content/post/kubedb-v2022.12.28/index.md [hunk 4]
```diff
@@ -0,0 +1,389 @@
...
+Like other KubeDB managed databases from now you will have autoscaling support for ProxySQL.
+
+We support compute (to autoscale CPU & memory resources) autoscaling currently. The structure of those autoscaler yamls is similar for all databases. Compute utilizes the VerticalScale OpsRequest internally while autoscaling. There are two more common fields in the CRDs spec :
+
+spec.proxysql to refer to the actual proxysql server.
+spec.opsRequestOptions to control the behavior of the ops-manager operator. It has two fields: apply and timeout.
+
+The supported values of spec.opsRequestOptions.apply are IfReady & Always. Use IfReady if you want to process the opsReq only when the proxysql server is Ready. And use Always if you want to process the execution of opsReq irrespective of the proxysql server state. spec.opsRequestOptions.timeout specifies the maximum time for each step of the opsRequest(in seconds). If a step doesn’t finish within the specified timeout, the ops request will result in failure.
+
+NB: We have also added Arbiter & HiddenNode support in MongoDB Autoscaler. We refer to the hidden-node corresponding section in MongoDB below, for that.
```
Comment (ArnobKumarSaha) [inline content/post/kubedb-v2022.12.28/index.md:137]: Why this colaveryD here ?

File: content/post/kubedb-v2022.12.28/index.md [hunk 5]
```diff
@@ -0,0 +1,389 @@
...
+        cpu: "0"
+```
+
+**Autoscaler:**
+Like other KubeDB managed databases from now you will have autoscaling support for ProxySQL.
+
+We support compute (to autoscale CPU & memory resources) autoscaling currently. The structure of those autoscaler yamls is similar for all databases. Compute utilizes the VerticalScale OpsRequest internally while autoscaling. There are two more common fields in the CRDs spec :
+
+spec.proxysql to refer to the actual proxysql server.
+spec.opsRequestOptions to control the behavior of the ops-manager operator. It has two fields: apply and timeout.
```
Comment (ArnobKumarSaha) [inline content/post/kubedb-v2022.12.28/index.md:133]: all the yaml fields should be inside backtick mark , like this : `spec.proxysql` to differentiate them from general words/sentences.

File: content/post/kubedb-v2022.12.28/index.md [hunk 6]
```diff
@@ -0,0 +1,389 @@
...
+- kubedb
+- kubernetes
+- mariadb
+- memcached
+- mongodb
+- mysql
+- opensearch
+- percona
+- percona-xtradb
+- postgresql
```
Comment (ArnobKumarSaha) [inline content/post/kubedb-v2022.12.28/index.md:23]: Remove unused tags.  For example: redis, memcachedb, schema-manager etc.

**Review by ArnobKumarSaha — CHANGES_REQUESTED** [review]

**Pull request merged**